bitnami/zookeeper/templates/metrics-svc.yaml

```diff
@@ -21,6 +21,9 @@ spec:
     - name: tcp-metrics
       port: {{ .Values.metrics.service.port }}
       targetPort: metrics
+      {{- if .Values.metrics.service.appProtocol }}
+      appProtocol: {{ .Values.metrics.service.appProtocol }}
+      {{- end }}
```

Comment:
Can't we simply rename the port name to `http-metrics`?

Reply:
Yep! That works as well, we would just need to update the port name for the ServiceMonitor to match. 

I considered that approach but I thought that might be too big of a lift and potentially break some deployments. What are your thoughts?

Reply:
What deployments do you refer to? I don't think other than metrics collector such as Prometheus would be interested on connecting to Zookeeper metrics service port. And, given we'll be adapting the ServiceMonitor in the same PR, I don't expect breaking changes.
In any case, you can always test your changes by running a `helm upgrade` after applying your changes and see if you find any errors.

Reply:
I was thinking if someone was deploying/managing the ServiceMonitor outside of the helm chart they might not pick up on that port name change. That would probably be a rare case though.

I'd prefer to change the port name to solve this problem anyways so if it's good with you then I can implement that!

Reply:
Yes please, change the port name so the `metrics.service.appProtocol` parameter isn't required.

Reply:
Made those updates and updated the PR title/description to reflect that. Thanks for the help!